packages/terra-form-input/tests/wdio/form-input-spec.js

```diff
@@ -145,6 +145,12 @@ describe('Form-Input', () => {
       });
     });
 
+    describe('RefCallback', () => {
+      beforeEach(() => browser.url('/#/raw/tests/terra-form-input/form-input/input-ref-callback'));
+
+      Terra.should.matchScreenshot({ viewports });
```

Comment:
This screenshot test doesn't seem to be proving the refcallback is triggered or valid. Can we add some sort on-screen text change or visual different that show the callback was called?

Reply:
I think the visual test should be removed in favor of a mocked jest test. Something similar to:
```
  it('should call refCallback', () => {
    const mockRefCallback = jest.fn();
    const wrapper = shallow(<Input refCallback={mockRefCallback} />);

    expect(mockRefCallback).toBeCalled();
  });
```

Reply:
The blue border is the visual indication. When the page loads, it auto focuses to the given input, causing the blue border to appear.

I'd also be more favored as well towards the jest test.

Reply:
To someone unfamiliar with the test, a blue border is not a clear indicator of what is happening here. I likely would have assumed focus was applied. I also agree testing via jest seems appropriate here.